index.html

```diff
@@ -258,5 +262,53 @@ <h2>
     </section>
 
     <!-- About Section Ends -->
+
+    <!-- Contact Me Form Starts -->
+
+    <section class="contact-me">
+      <h2 class="contact-me-tittle">Contact Me</h2>
```

Comment:
- [ ] Kindly make sure that the title is centered to match the Figma design.


| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/221903863-0ad4bcf3-dd8f-4d14-bdb4-c21396a13253.png) | ![image](https://user-images.githubusercontent.com/62290545/221904018-5a0d861f-b290-42a7-87b3-a5be5e34c368.png) |

Reply:
Done!